modules/nf-core/merfin/hist/meta.yml

```diff
@@ -61,14 +61,19 @@ output:
           description: |
             Groovy Map containing sample information
             e.g. `[ id:'sample1', single_end:false ]`
+          pattern: "*.{hist}"
       - "*.hist":
-          type: file
-          description: The generated 0-centered k* histogram for sequences in <fasta_assembly.fasta>.
-            Positive k* values are expected collapsed copies. Negative k* values are expected
-            expanded  copies. Closer to 0 means the expected and found k-mers are well
-            balenced, 1:1.
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
```

Comment:
What happened here?

Reply:
I didn't check the results of `--fix`. Thanks.